source/common/tracing/opentracing_driver_impl.cc

```diff
@@ -164,8 +170,13 @@ SpanPtr OpenTracingDriver::startSpan(const Config&, Http::HeaderMap& request_hea
       tracerStats().span_context_extraction_error_.inc();
     }
   }
-  active_span = tracer.StartSpan(operation_name, {opentracing::ChildOf(parent_span_ctx.get()),
-                                                  opentracing::StartTimestamp(start_time)});
+  opentracing::StartSpanOptions options;
+  opentracing::ChildOf(parent_span_ctx.get()).Apply(options);
+  opentracing::StartTimestamp(start_time).Apply(options);
+  if (!tracing_decision.traced) {
+    opentracing::SetTag(OpenTracingTags::get().SAMPLING_PRIORITY, 0).Apply(options);
```

Comment:
I added a tag for the [sampling priority](https://github.com/opentracing/opentracing-cpp/blob/master/include/opentracing/ext/tags.h#L44) to match what's in the other APIs so you could write
```cpp
options.tags.emplace_back(opentracing::ext::sampling_priority, 0);
```

Reply:
Updated to use the new constant.